runtime/src/lib.rs

```diff
@@ -834,6 +883,10 @@ construct_runtime!(
         Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
         Proxy: pallet_proxy::{Module, Call, Storage, Event<T>},
 		Multisig: pallet_multisig::{Module, Call, Storage, Event<T>},
+		Registry: va_registry::{Module, Call, Storage, Event<T>},
+		Nft: nft::{Module, Call, Storage, Event<T>},
```

Comment:
Please add the new pallets always at the end, otherwise it can screw up the metadata indices